streaming/src/main/scala/org/apache/spark/streaming/scheduler/ReceivedBlockTracker.scala

```diff
@@ -40,7 +41,9 @@ private[streaming] case class BatchAllocationEvent(time: Time, allocatedBlocks:
   extends ReceivedBlockTrackerLogEvent
 private[streaming] case class BatchCleanupEvent(times: Seq[Time])
   extends ReceivedBlockTrackerLogEvent
-
+// Each event is in serialized format inside the array
+private[streaming] case class CombinedReceivedBlockTrackerLogEvent(
```

Comment:
This should not be needed any more as RBT should not need to be aware of batching any more.